README.md

```diff
@@ -13,13 +13,20 @@
   * [Paciello Group](https://developer.paciellogroup.com/resources/)
   * [Ebay accessibility patterns](https://ebay.gitbooks.io/mindpatterns/content/)
   * [BBC accessibility guidelines](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/)
+  * [Vox Media Accessibility Checklist](http://accessibility.voxmedia.com/)
 
-## Blogs, etc.
+## Other Resources (Blogs, Individuals, Videos, etc)
 
   * [24 Accessibility](https://www.24a11y.com/) - series of articles on digital accessibility
-  * [Leonie Watson](https://tink.uk/)
-  * [Scott O'Hara](https://www.scottohara.me/all-writings/)
+  * [Leonie Watson](https://tink.uk/) - Leading Accessibility engineer, on W3C Web Platform Working Group. [@LeonieWatson](https://twitter.com/LeonieWatson) 
+  * [Scott O'Hara](https://www.scottohara.me/all-writings/) UX Accessibility at Paciello Group. [@scottohara](https://twitter.com/scottohara)
   * [Alastair Campbell](https://alastairc.ac/)
   * [Inclusive Components](https://inclusive-components.design/) - pattern library of accessible web interfaces
-  * [A11ycasts with Rob Dodson](https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g) - series of videos on the fundamentals of building accessible apps by Rob Dodson, developer advocate at Google
+  * [Video: A11ycasts with Rob Dodson](https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g) - series of videos on the fundamentals of building accessible apps by Rob Dodson, developer advocate at Google
```

Comment:
Let's make Video its own heading, just like *Helpful Topical Reads*, then move Leonie Watson's video under that. We can remove the *Video* label from the links.